src/Agent.Worker/Release/ReleaseDefinitionToFolderMap.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    public sealed class ReleaseDefinitionToFolderMap
+    {
+        [JsonProperty("release_artifactsdirectory")]
+        public string ArtifactsDirectory { get; set; }
```

Comment:
This would be _work/1
But the artifacts directory would be _work/1/a, right?

Rename this to ParentWorkingDirectory?